drivers/hmis/app/models/hmis/ce/opportunity.rb

```diff
@@ -42,9 +42,8 @@ class Opportunity < GrdaWarehouseBase
       end
     end
 
-    # TODO(#7395): permissions
     scope :viewable_by, ->(user) do
-      joins(:project).merge(Hmis::Hud::Project.viewable_by(user))
+      joins(:project).merge(Hmis::Hud::Project.with_access(user, :can_view_units))
```

Comment:
Why did we drop the project viewability scope here? I think we are losing the check for can_view_projects by removing this.